internal/version/version.go

```diff
@@ -121,9 +121,9 @@ func (v PatchVersion) ToMinor() MinorVersion {
 	return MinorVersion(latestPatchFormat)
 }
 
-// patch attempts to parse a patch number from the Version.String.
+// Patch attempts to parse a Patch number from the Version.String.
 // This will always succeed when created via NewVersion or NewPatchVersion
-func (v PatchVersion) patch() int {
+func (v PatchVersion) Patch() int {
```

Comment:
exporting this to help in the linting is fine imho as it helps decouple online vs offline checks (ex we don't want unit tests to have to call out to func-e.io!)